security/advancedtls/advancedtls_test.go

```diff
@@ -43,7 +43,15 @@ func TestClientServerHandshake(t *testing.T) {
 	getRootCAsForClient := func(params *GetRootCAsParams) (*GetRootCAsResults, error) {
 		return &GetRootCAsResults{TrustCerts: clientTrustPool}, nil
 	}
-	verifyFuncGood := func(params *VerificationFuncParams) (*VerificationResults, error) {
+	clientVerifyFuncGood := func(params *VerificationFuncParams) (*VerificationResults, error) {
+		if params.ServerName == "" {
+			return nil, fmt.Errorf("client side server name should have a value")
```

Comment:
Prefer using `errors.New` instead of `fmt.Errorf` since there are not formatting verbs in the format string.